xmppserver/src/main/java/org/jivesoftware/openfire/streammanagement/StreamManager.java

```diff
@@ -269,6 +269,11 @@ private void startResume(String namespace, String previd, long h) {
             sendError(new PacketError(PacketError.Condition.unexpected_request));
             return;
         }
+        if (otherSession.getStreamManager().namespace == null) {
+            Log.debug("Not allowing a client to resume a session, the session to be resumed disabled SM functionality as a response to an earlier error." );
```

Comment:
could any details about the client be included in this message or does that automagically happen already with the logger?

Reply:
That's a good suggestion. I'll add that.